### ISSUE TITLE
Добавить возможность перехода к ссылкам, формированию табеля и карточке сотрудника прямо из документов Начисление ЗП и Выплата ЗП

### ISSUE THREAD

**Issue by Nullarity** [Contabilizare/c5#241]
Хотелось бы, чтобы по правому клику на выбранном сотруднике, можно было сразу перейти к одному из перечисленных объектов.

**Comment by Linda4ka** [comment]
Можно уточнение: к формированию табеля из документа Начисление зп? Табель же формируется до начисления зарплаты, или я неправильно поняла?

**Comment by Nullarity** [comment]
да, для оперативного получения информации по отработанному времени в случае необходимости (проверить или скорректировать расчеты программы)
